Update adding_metadata.md (#24196)
DataDog/documentation: content/en/service_catalog/adding_metadata.md

This GitHub Action allows you to register your services with the Service Catalog using a GitHub Action, with full control over when this information is sent to Datadog, and implement other compliance checks unique to your organization.

## IDE Plugins

> Datadog provides a [JSON Schema][17] for service definitions so that when you are editing a service definition in a [supporting IDE][18], features such as autocomplete and validation are provided. The [JSON schema for Datadog service definitions][19] is registered with the open source [Schema Store][18].

Reviewer: ```suggestion
Datadog provides a [JSON Schema][17] for service definitions so you can use features like autocomplete and validation when editing a service definition in a [supported IDE][18]. The [JSON Schema for Datadog service definitions][19] is registered with the open source [Schema Store][18].
```

> The plugin supports the new v3.0 schema and includes information that will tell your IDEs which filenames it applies to—specifically, the files entity.datadog.yaml, entity.datadog.yml, and entity.datadog.json to quickly and accurately complete your entity definition file without referencing documentation. In the example below, the IDE automatically creates keys as you fill out the entity definition.

Reviewer: We say "plugin" here (and "plugin" is in the title as well), but it's unclear to me what exactly this refers to. Is the Datadog JSON Schema a plugin? I would suggest using plugin everywhere or using some other word like "integration". Whatever you think is best, and just stay consistent.

Reviewer: Also, I'm not sure how much value-add it is to include "supports the new v3.0 schema"? If you do think this is important, then I'd ask you to clarify what this refers to -- is v3.0 the version for JSON Schema or for Datadog's JSON Schemas? (Let me know if my question doesn't make sense.)

> The plugin supports the new v3.0 schema and includes information that will tell your IDEs which filenames it applies to—specifically, the files entity.datadog.yaml, entity.datadog.yml, and entity.datadog.json to quickly and accurately complete your entity definition file without referencing documentation. In the example below, the IDE automatically creates keys as you fill out the entity definition.

Reviewer: ```suggestion
The plugin informs your IDE about the applicable files—namely, `entity.datadog.yaml`, `entity.datadog.yml`, and `entity.datadog.json`—so you can complete your entity definition file without referencing documentation. In the following example, the IDE automatically creates keys as you fill out the entity definition.
```

> {{< img src="tracing/service_catalog/service_catalog_ide.mp4" video=true alt="Using the IDE plugin to atucomplete and validate definitions" style="width:100%;" >}}

Reviewer: ```suggestion
{{< img src="tracing/service_catalog/service_catalog_ide.mp4" video=true alt="A developer using the IDE plugin to autocomplete and validate definitions" style="width:100%;" >}}
```

[14]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/service_definition_yaml
[15]: https://github.com/DataDog/schema/blob/main/service-catalog/v2.2/schema.json
[16]: https://github.com/DataDog/schema/tree/main/service-catalog/v3
[17]: http://json-schema.org/
[18]: https://www.schemastore.org/json/

> [19]: https://raw.githubusercontent.com/DataDog/schema/main/service-catalog/version.schema.json

Reviewer: This link seems to be broken? I'm getting a 404.

Reviewer: Is this the right link: https://github.com/DataDog/schema/blob/main/service-catalog/service.schema.json ??
